.env

```diff
@@ -12,7 +12,7 @@ FLUTTER_EXAMPLE_WORKSPACE=discovery_engine_flutter/example
 DART_VERSION=2.17.3
 FLUTTER_VERSION=3.0.2
 
-ANDROID_TARGETS="arm64-v8a x86_64 x86"
+ANDROID_TARGETS="aarch64-linux-android x86_64-linux-android i686-linux-android"
```

Comment:
our github actions also contain the `armv7-linux-androideabi` target. we should either add it here as well or remove it there, what's the current situation?

Reply:
We usually don't want to build for it locally while we want it on the CI.

Reply:
ah ok, then i'll leave it as is 👌